src/main/java/uk/gov/homeoffice/digital/sas/timecard/kafka/exceptions/KafkaInterruptedException.java

```diff
@@ -0,0 +1,7 @@
+package uk.gov.homeoffice.digital.sas.timecard.kafka.exceptions;
+
+public class KafkaInterruptedException extends InterruptedException {
```

Comment:
Is this used anywhere?

Reply:
Accidentally left in from previous commit removed